library/src/main/java/me/grantland/widget/AutofitHelper.java

```diff
@@ -232,8 +262,9 @@ else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
     private float mMaxTextSize;
     private float mPrecision;
 
-    private boolean mEnabled;
+    private boolean mIsAutofitWidthEnabled;
     private boolean mIsAutofitting;
+    private static boolean mIsAutofitHeightEnabled;
```

Comment:
I think you could leave the method `autofit(...)` static, but then pass `mIsAutofitHeightEnabled` as a parameter to it. This way, each `AutofitTextView` can have its own `mIsAutofitHeightEnabled` setting instead of sharing a global one.